ScoutSuite/providers/gcp/rules/rulesets/cis-1.1.0.json

```diff
@@ -6,7 +6,19 @@
               "enabled": true,
               "level": "warning"
           }
-      ]
+      ],
+      "computeengine-network-default-in-use.json": [
+            {
+                "enabled": true,
+                "level": "danger"
```

Comment:
Better put `warning` as we're flagging the actual firewall rules independently.

Reply:
I've changed it to warning and also added to the partial to show if the network is legacy or not